ci/infra/vmware/lb-instance.tf

```diff
@@ -136,6 +136,7 @@ resource "vsphere_virtual_machine" "lb" {
   resource_pool_id = data.vsphere_resource_pool.pool.id
   datastore_id     = (var.vsphere_datastore == null ? null: data.vsphere_datastore.datastore[0].id)
   datastore_cluster_id = (var.vsphere_datastore_cluster == null ? null : data.vsphere_datastore_cluster.datastore[0].id)
+  folder           = var.setup_cloud_provider == true ? "${var.stack_name}-cluster" : null
```

Comment:
Should we also add ` depends_on       = [vsphere_folder.folder]` here like master/worker?

Reply:
I've thought about it, but since lb already have `depends_on = [vsphere_virtual_machine.master]`. IMO it should not need extra dependency.

Reply:
Technically yes, but actually having explicit statements makes the code clear to understand and dependency resolving will be handled by terraform itself. But this is minor actually :)

Reply:
Done.